lib/Runtime/Types/TypePropertyCache.cpp

```diff
@@ -118,7 +118,7 @@ namespace Js
         Assert(prototypeObjectWithProperty);
 
         const TypePropertyCacheElement &element = elements[ElementIndex(id)];
-        if(element.Id() != id || !checkMissing && element.IsMissing())
+        if(element.Id() != id || (!checkMissing && element.IsMissing()))
```

Comment:
Edited this one in order to get rid of a clang warning. Hoping the location for parentheses are correct?